stable/grafana/templates/deployment.yaml

```diff
@@ -122,6 +122,17 @@ spec:
                   name: {{ .Values.smtp.existingSecret }}
                   key: password
             {{- end }}
+            {{- if .Values.database.existingSecret }}
+            - name: GF_DATABASE_USER
```

Comment:
Couldn't this be implemented using the changes from https://github.com/kubernetes/charts/pull/5435?